SCHEMA_IR_POC.md

```
@@ -0,0 +1,139 @@
+interface FormischFieldIR {
+  readonly type: 'object' | 'array' | 'string' | 'number' | 'boolean' | 'date' | 'bigint' | 'unknown';
+  readonly optional: boolean;
+  getDefault(): unknown;
+  readonly properties?: ReadonlyMap<string, FormischFieldIR>;
+  readonly item?: FormischFieldIR;
+}
+```
+
+### Data Flow
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)